examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -24,17 +22,7 @@ class MainApplication : Application(), UpdatedCustomerInfoListener {
     override fun onCreate() {
         super.onCreate()
 
-        if (BuildConfig.DEBUG) {
-            StrictMode.setVmPolicy(
-                VmPolicy.Builder()
-                    .detectLeakedClosableObjects()
-                    .penaltyLog()
```

Comment:
Would it be worth to keep the `Log`? Though I guess until we discover why the current issue is happening, we would kinda ignore it, so this makes sense for now